modules/oe_theme_helper/tests/src/Unit/TwigExtensionTest.php

```diff
@@ -262,18 +262,20 @@ protected static function getEuropeanUnionLanguageList(): array {
    *
    * @param string $icon_name
    *   The icon name.
+   * @param string|null $size
+   *   The icon size.
    * @param array $expected_icon_array
    *   The icon array to be rendered.
    *
    * @covers ::toEclIcon
    * @dataProvider toEclIconProvider
    */
-  public function testToEclIcon(string $icon_name, array $expected_icon_array) {
+  public function testToEclIcon(string $icon_name, string $size = NULL, array $expected_icon_array) {
```

Comment:
About `string $size = NULL,` let's move optional parameters at the end, after all the mandatory ones. Or, if we want to keep it here, let's remove `= NULL`, it's a test anyway, we control what's passed to this method in `toEclIconProvider`